pkg/throttle/writer.go

```diff
@@ -0,0 +1,37 @@
+package throttle
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"golang.org/x/time/rate"
+)
+
+type writer struct {
+	limiter *rate.Limiter
+	writer  io.Writer
+}
+
+func NewWriter(w io.Writer, br Byterate) io.Writer {
+	burstLimit := 1024 * 1024 * 1024
```

Comment:
Kan we make `1024*1024*1024` a const with a descriptive name?

Reply:
I agree.